scripts/generators/stubGenerator.sh

```diff
@@ -0,0 +1,333 @@
+# This script generates a stub from a given interface
+# Mandatory parameters needed:
+#   interface name from an interface.go file
+#   path to the directory of interface.go file, from the directory this script is called
+#   path to the destination directory the stub will be created, from the directory this script is called
+#
+# Usage example: bash stubGenerator.sh EnableEpochsHandler ../../common ../../common
+
+generateStub() {
+  interfaceName=$1
+  filePath=$2"/interface.go"
+  stubDir=$3
+
+  [ ! -d "$2" ] && echo "Source directory for interface DOES NOT exists." && exit
+  [ ! -f "$filePath" ] && echo "Source interface.go file DOES NOT exists." && exit
+  [ ! -d "$stubDir" ] && echo "Destination directory DOES NOT exists." && exit
+
+  extractPackageName
```

Comment:
The function definition must be placed before any calls to the function !!! (bash is not a compiler)